lib/active_resource/railtie.rb

```diff
@@ -10,6 +10,14 @@ class Railtie < Rails::Railtie
         ActiveResource::Base.send "#{k}=", v
       end
     end
+
+    initializer "active_resource.add_active_job_serializer" do |app|
+      ActiveSupport.on_load(:active_job) do
```

Comment:
Don't need the on_load anymore. We are just setting the config, we are not loading to active job.